src/System.Console/src/System/IO/SyncTextReader.cs

```diff
@@ -31,8 +31,8 @@ protected override void Dispose(bool disposing)
             {
                 lock (_methodLock)
                 {
-                    // Explicitly pick up a potentially methodimpl'ed Dispose
-                    ((IDisposable)_in).Dispose();
+                    // This will always pick up TextReader.Dispose
```

Comment:
> I'm not sure why we'd be using this pattern to pick up the custom `IDisposable` implementation here but not elsewhere.

Technically this is the behavior of a `using` statement so it gets used frequently.